webpack/webpack.common.js

```diff
@@ -0,0 +1,49 @@
+const path = require("path");
+const HtmlWebpackPlugin = require("html-webpack-plugin");
+
+module.exports = {
+    // entry: {
+    //     main: "./src/index.js",
```

Comment:
Please, delete comments. Code in the master branch should contain only clarification comments or JSDocs.